bitnami/phabricator/values.yaml

```diff
@@ -86,99 +134,258 @@ phabricatorGitSSH: false
 # smtpPassword:
 # smtpProtocol:
 
+## Command and args for running the container (set to default if not set). Use array form
 ##
-## MariaDB chart configuration
+command: []
+args: []
+
+## An array to add extra env vars
+## Example:
+## extraEnvVars:
+##   - name: FOO
+##     value: "bar"
 ##
-## https://github.com/bitnami/charts/blob/master/bitnami/mariadb/values.yaml
+extraEnvVars: []
+
+## ConfigMap with extra environment variables
 ##
-mariadb:
-  ## Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database set this to false and configure the externalDatabase parameters
+extraEnvVarsCM:
+
+## Secret with extra environment variables
+##
+extraEnvVarsSecret:
+
+## Strategy to use to update Pods
+##
+updateStrategy:
+  ## StrategyType
+  ## Can be set to RollingUpdate or OnDelete
   ##
+  type: RollingUpdate
+
+## Phabricator container ports to open
+##
+containerPorts:
+  http: 8080
+  https: 8443
+  ssh: 2222
+
+## Phabricator pods' Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+##
+podSecurityContext:
   enabled: true
+  fsGroup: 1001
 
-  ## MariaDB architecture. Allowed values: standalone or replication
+## Phabricator containers' SecurityContext
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+##
+containerSecurityContext:
+  enabled: true
+  runAsUser: 1001
+
+## Phabricator resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits: {}
+  #   cpu: 200m
+  #   memory: 256Mi
+  requests:
+    memory: 512Mi
+    cpu: 300m
+
+## Phabricator containers' liveness and readiness probes.
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+##
+livenessProbe:
+  enabled: true
+  httpGet:
+    path: /auth/
+    port: http
+    httpHeaders:
+      - name: Host
+        value: "{{ include \"phabricator.host\" . }}"
+  initialDelaySeconds: 180
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+readinessProbe:
+  enabled: true
+  httpGet:
+    path: /auth/
+    port: http
+    httpHeaders:
+      - name: Host
+        value: "{{ include \"phabricator.host\" . }}"
+  initialDelaySeconds: 30
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
```

Comment:
Sorry I cannot find in that thread that we agreed on only adding the specific fields..
Also, IMHO all the parameters included there are likely to be cuztomizable.

Reply:
It seems like we discussed internally, just noticed that, sorry. The thing is that we do not want to be 100% flexible with `livenessProbe/readinessProbe`, since that's what `customLivenessProbe/customReadinessProbe` is for.

Check this issue/PR for more info: #2022

Also ccing @javsalgar in case you have questions about this.